packages/editor-ui/src/components/CodeNodeEditor/CodeNodeEditor.vue

```diff
@@ -156,34 +151,52 @@ export default mixins(linterExtension, completerExtension, workflowHelpers).exte
 				if (!viewUpdate.docChanged) return;
 				this.trackCompletion(viewUpdate);
 				this.$emit('valueChanged', this.content);
+				if (this.content === this.defaultValue) {
+					this.isDefault = true;
+				} else {
+					this.isDefault = false;
+				}
 			}),
 		];
 
 		// empty on first load, default param value
-		if (this.code === '') {
-			this.$emit('valueChanged', this.placeholder);
+		if (this.value === '') {
+			this.$emit('valueChanged', this.defaultValue);
 		}
 
+		this.isDefault = this.value === this.defaultValue;
+
+		const value = this.value === '' ? this.defaultValue : this.value;
```

Comment:
nitpick - maybe rename this var from `value` for clarity? `resolvedValue`
```suggestion
		const value = this.value === '' ? this.defaultValue : this.value;
```

Reply:
Thanks a lot. Got changed.